src/components/UserList/UserListToolbar.tsx

```diff
@@ -35,8 +32,9 @@ export function UserListToolbar(props: UserListToolbarProps) {
       )}
       {numSelected > 0 ? (
         <>
-          {selectedRows.every((row) => {
-            return row.userType === 'Pending';
+          {selected.every((row) => {
+            /*back doesn't have a pending type yet, this will need to be changed later*/
+            return row.roles?.at(0)?.id === UserTypesId.PENDING;
```

Comment:
I think in backend we do not have `PENDING` and empty array means that user is in PENDING state. I think we have discussed this last time. Or we have decided on sth else? Sorry, I do not remember